package.json

```diff
@@ -55,12 +56,12 @@
     "@xterm/addon-canvas": "0.7.0",
     "@xterm/xterm": "5.5.0",
     "dequal": "2.0.3",
+    "path-browserify": "^1.0.1",
```

Comment:
No hats please, we want to pin down versions.

Reply:
Yes, done.